src/lib/components/Swap/Summary/Details.tsx

```diff
@@ -27,31 +28,42 @@ function Detail({ label, value }: DetailProps) {
 }
 
 interface DetailsProps {
-  input: Currency
-  output: Currency
+  trade: Trade<Currency, Currency, TradeType>
 }
 
-export default function Details({ input, output }: DetailsProps) {
+export default function Details({ trade }: DetailsProps) {
   const integrator = window.location.hostname
 
-  const { maxSlippage } = useAtomValue(settingsAtom)
+  const { allowedSlippage } = useSwapInfo()
```

Comment:
Should this be passed through props as well?

Reply:
yeah i think so - either all grab from state or make this presentational - ill update for the latter